src/test/kotlin/com/bardsoftware/papeeria/backend/cosmas/CosmasGoogleCloudServiceTest.kt

```diff
@@ -483,6 +484,51 @@ class CosmasGoogleCloudServiceTest {
         assertEquals(newPatch("-4", "patch4", 4), list2[1])
     }
 
+    @Test
+    fun deleteFile() {
+        val fakeStorage = mock(Storage::class.java)
+        val projectId = "1"
+        val fileId = "1"
+        val time = 1L
+        val cemetery = FileCemetery.newBuilder().addAllCemetery(mutableListOf()).build()
```

Comment:
nit: you may use `listOf` here, it doesn't need to be mutable

Reply:
done